### ISSUE TITLE
"noresults" slot not working

### ISSUE THREAD

**Issue by Ajwang002** [vueform/multiselect#393]
<template #noresults>
    <div class="d-flex justify-center my-3 text-body-2">
      {{ contact_type }} Not Found 🚫.
    </div>
</template>

 <template #nooptions>
    <div class="d-flex justify-center my-3 text-body-2">
      The {{ contact_type }} list is empty 🚫.
    </div>
</template>

The "nooptions" slot is functioning as expected, but the "noresults" slot is not working in the given scenario.

**Comment by adamberecz** [comment]
Would closing the first `<template>` tag help?

**Comment by Ajwang002** [comment]
> Would closing the first `<template>` tag help?

Yeah I closed 'noresults' template tag, even also it's not working...

**Comment by adamberecz** [comment]
It does on my end, so feel free to create a reproduction and I'll have a look.
